service/CapabilityService.ts

```diff
@@ -0,0 +1,13 @@
+const axios = require('axios');
+import { Capability } from "../model/Capability";
+
+
+module.exports.getCapabilities = async function() {
+    try {
+        const response = await axios.get('http://localhost:8080/api/capabilities')
```

Comment:
pass 'http://localhost:8080' through as an environment variable for we can set it to the AWS URL when deployed

Reply:
looks like this is already done in job role service: process.env.UI_URL